silkworm/node/stagedsync/stages/stage_senders.cpp

```diff
@@ -240,6 +240,12 @@ Stage::Result Senders::prune(db::RWTxn& txn) {
 }
 
 Stage::Result Senders::parallel_recover(db::RWTxn& txn) {
+
+    // secp256k1_context is thread safe and can be reused, but the creation is expensive (~10ms)
+    static secp256k1_context* context = secp256k1_context_create(SILKWORM_SECP256K1_CONTEXT_FLAGS);
+    // release only when the process exits
+    static auto _ = gsl::finally([&]() { if (context) std::free(context); });
```

Comment:
I know this is just moving existing code, and while `free()` might be fine it seems like this really ought to be `secp256k1_context_destroy()`

Reply:
yes you're right